testfixtures/tests/test_compare.py

```diff
@@ -150,28 +312,71 @@ def test_sequence_long(self):
              'quite a long string 3','quite a long string 4',
              'quite a long string 9','quite a long string 10',
              'quite a long string 11','quite a long string 12'],
-            "sequence not as expected:\n\n"
-            "same:\n"
-            "['quite a long string 1',\n"
-            " 'quite a long string 2',\n"
-            " 'quite a long string 3',\n"
-            " 'quite a long string 4']\n\n"
-            "first:\n"
-            "['quite a long string 5',\n"
-            " 'quite a long string 6',\n"
-            " 'quite a long string 7',\n"
-            " 'quite a long string 8']\n\n"
-            "second:\n"
-            "['quite a long string 9',\n"
-            " 'quite a long string 10',\n"
-            " 'quite a long string 11',\n"
-            " 'quite a long string 12']\n"
-            "\n"
-            "While comparing [4]: \n"
-            "'quite a long string 5'\n"
-            "!=\n"
-            "'quite a long string 9'"
-            )
+            dedent("""\
+                sequence not as expected:
+
+                same:
+                ['quite a long string 1',
+                 'quite a long string 2',
+                 'quite a long string 3',
+                 'quite a long string 4']
+
+                first:
+                ['quite a long string 5',
+                 'quite a long string 6',
+                 'quite a long string 7',
+                 'quite a long string 8']
+
+                second:
+                ['quite a long string 9',
+                 'quite a long string 10',
+                 'quite a long string 11',
+                 'quite a long string 12']
+
+                While comparing [4]: 
+                'quite a long string 5'
+                !=
+                'quite a long string 9'"""))
```

Comment:
sorry, but I prefer the original style, please can you leave these changes out of the pull request.